src/main/java/uk/gov/hmcts/ccd/domain/model/search/UseCase.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.ccd.domain.model.search;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.base.Strings;
+
+import java.util.Arrays;
+
+public enum UseCase {
+
+    WORKBASKET("WORKBASKET"),
+    SEARCH("SEARCH"),
+    ORG_CASES("ORGCASES"),
+    DEFAULT("");
```

Comment:
temporary: does this mean every time a service introduces a new use case we need to do a new release of Data Store?